src/neural/onnx/converter.cc

```diff
@@ -1028,6 +1027,13 @@ void Converter::Convert(pblczero::Net* dst) {
 
 }  // namespace
 
+WeightsToOnnxConverterOptions::DataType
+WeightsToOnnxConverterOptions::StringToDataType(const std::string& s) {
+  if (s == "f32") return DataType::kFloat32;
+  if (s == "f16") return DataType::kFloat16;
+  throw Exception("Invalid data type: " + s);
```

Comment:
Maybe add the accepted types in the message? This will help fix wrong values in the `data_type` backend option e.g. if fp16 or fp32 are used.